### ISSUE TITLE
BN9 "without using hacknet servers" problem

### ISSUE THREAD

**Issue by Orpheusolaf** [danielyxie/bitburner#4142]
Hello
I'm in BN9.4 (bitburner version v2.0.2)/. 
I'm trying to destroy BN9 "without using hacknet servers". But when I backdoor the w0r1d_d43m0n, without buying a single hacknet, I get to the Bitverse but without the achievement.
Have you already achieved this goal or am I the only one with this problem?
Thxxx

**Comment by hydroflame** [comment]
Did make any money using hacknet servers or purchase any hacknet servers?

**Comment by Orpheusolaf** [comment]
Nope. I didn't buy a single hacknet server...

**Comment by hydroflame** [comment]
Upload your save file.

**Comment by Orpheusolaf** [comment]
It doesn't support JSON format.
What can I do?

Le ven. 23 sept. 2022, 18:51, hydroflame ***@***.***> a
écrit :

**Comment by hydroflame** [comment]
zip it first.

**Comment by Orpheusolaf** [comment]
[bitburnerSave_1663947570_BN9x4_WithoutHacknet.zip](https://github.com/danielyxie/bitburner/files/9646383/bitburnerSave_1663947570_BN9x4_WithoutHacknet.zip)

**Comment by Snarling** [comment]
Looks like you used hacknet
![image](https://user-images.githubusercontent.com/84951833/192294904-fcc79173-abff-4d37-bee9-f796149e111f.png)

**Comment by Orpheusolaf** [comment]
indeed.
I don't know how. I thought I was being careful.
I start again
Thxx

**Comment by Orpheusolaf** [comment]
Hello
I understand now the problem.
I have already passed the BN9.3. Dus, now I start BN9 with a hacknet server working...
It seems not more possible to get this achievement...

**Comment by Herzfinsternis** [comment]
The problem is, the server is going to generate money, when the cache is full - it seems... So, start BN9 and then softreset to get rid of that server before the cache is full.

![Clipboard01](https://user-images.githubusercontent.com/25349375/194519907-4487294b-9f59-4ddf-98da-05bbba0b92c0.png)

![Clipboard02](https://user-images.githubusercontent.com/25349375/194519931-33517190-4f04-4229-999c-2faa08b77486.png)

**Comment by Orpheusolaf** [comment]
Since BN9.3. grants a "highly-upgraded Hacknet Server when entering a new BitNode", I don't see how to get the achievements.

**Comment by Herzfinsternis** [comment]
> Since BN9.3. grants a "highly-upgraded Hacknet Server when entering a new BitNode", I don't see how to get the achievements.

I did a test after importing your savegame. Ran b1t_flum3.exe and started BN9. Softreset to get rid of that hacknet server, hacked w0r1d_d43m0n and finally got the achievement.

![Clipboard01](https://user-images.githubusercontent.com/25349375/194545004-371d6f3b-7783-45ab-ba94-70726300313a.png)

![Clipboard02](https://user-images.githubusercontent.com/25349375/194545046-29f234f2-11cf-437c-a155-34f4a28b28b8.png)

Voilà!
